Receipts/ReceiptListViewController.swift

```diff
@@ -1,33 +1,24 @@
 import UIKit
 
-class ReceiptListViewController: UIViewController, UITableViewDataSource, UITableViewDelegate, NewReceiptFormViewControllerDelegate {
+class ReceiptListViewController: UIViewController, NewReceiptFormViewControllerDelegate {
     
     var receiptStore: ReceiptStore!
-    var receipts: [Receipt] {
+    private var receipts: [Receipt] {
         return receiptStore.receipts
     }
     
-    let remoteStore = ReceiptRemoteStore()
-    var saveNotificationObserver: NSObjectProtocol?
+    var tableViewManager: ReceiptTableViewManager!
     
     @IBOutlet var tableView: UITableView!
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        tableView.allowsMultipleSelectionDuringEditing = false
+        tableViewManager = ReceiptTableViewManager(tableView: tableView, receiptStore: receiptStore)
```

Comment:
If this were more than one line or the `viewDidLoad` had more than just this call, I would have made a private method below and called it.